src/Bubble/README.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Bubble'
+type: 'component'
+components:
+- Bubble
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+---
+
+Represents the filled circle with a number of an icon, supporting the usual color variants
+
+### Basic Usage
```

Comment:
Should there be an example of using the `disabled` prop and what that looks like?

Reply:
Yes, I think it is good idea to separate `disabled` from other `variants`